src/main/java/org/kainos/ea/db/EmployeeDao.java

```diff
@@ -0,0 +1,31 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.EmployeeRequest;
+
+import java.sql.*;
+
+public class EmployeeDao {
+    private final DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public int createEmployee(EmployeeRequest employee) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO Employee (Name, Salary, Bank_Acc_Number, NI_Number)\n" +
```

Comment:
How will the query handle `\n`?

Reply:
It handles as a new line in the query and should work fine but I'll remove it